.github/workflows/ci.yml

```
@@ -27,7 +27,7 @@ jobs:
       # This line enables distribution
       # The "--stop-agents-after" is optional, but allows idle agents to shut down once the "e2e-ci" targets have been requested
       # - run: npx nx-cloud start-ci-run --distribute-on="3 linux-medium-js" --stop-agents-after="e2e-ci"
-      - run: npm ci
+      - run: yarn install --frozen-lockfile 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)